src/views/CheckoutTransmission.vue

```diff
@@ -3,12 +3,10 @@
         <CheckoutDetails :accountChangeable="false"/>
         <Network ref="network" :alwaysVisible="true" message="Sending transaction"/>
         <transition name='fade-in'>
-            <div class="success center" v-if="isTxSent">
-                <div class="icon-checkmark-circle"></div>
-                <h1>Your payment was successful!</h1>
-                <div style="flex-grow: 1;"></div>
-                <button @click="done">Back to store</button>
-            </div>
+            <Success v-if="isTxSent"
+                successText="Your payment was successful"
+                v-bind:appName="this.keyguardRequest.appName"
+                v-bind:close="this.done" />
```

Comment:
When changing `close` to be an event in `Success.vue`, this becomes `@close="done"`.